src/server/cluster/cluster_family.cc

```diff
@@ -425,12 +425,8 @@ void ClusterFamily::DflyCluster(CmdArgList args, ConnectionContext* cntx) {
   return cntx->SendError(UnknownSubCmd(sub_cmd, "DFLYCLUSTER"), kSyntaxErrType);
 }
 
-void ClusterFamily::DflyClusterMyId(CmdArgList args, ConnectionContext* cntx) {
-  if (!args.empty()) {
-    return cntx->SendError(WrongNumArgsError("DFLYCLUSTER MYID"));
-  }
```

Comment:
Why did you remove the arguments check?

Reply:
To be consistent with other CLUSTER commands

Reply:
But this command doesn't accept parameters..
Please do apply this check :)

Reply:
I will add it to other cluster commands in this case